packages/datagateway-common/src/card/cardView.component.tsx

```diff
@@ -615,14 +642,35 @@ const CardView = (props: CardViewProps): React.ReactElement => {
                             >
                               <ListItemText primary={s.label} />
                               <ListItemIcon>
-                                <TableSortLabel
-                                  active={s.dataKey in sort}
-                                  direction={sort[s.dataKey]}
-                                  // Set tabindex to -1 to prevent button focus
-                                  tabIndex={-1}
-                                >
-                                  {s.dataKey in sort && sort[s.dataKey]}
-                                </TableSortLabel>
+                                {s.dataKey in sort ? (
+                                  <TableSortLabel
+                                    active={true}
+                                    direction={sort[s.dataKey]}
+                                    // Set tabindex to -1 to prevent button focus
+                                    tabIndex={-1}
+                                  >
+                                    {s.dataKey in sort && sort[s.dataKey]}
+                                  </TableSortLabel>
+                                ) : shiftDown ? (
+                                  <TableSortLabel
+                                    active={true}
+                                    direction={sort[s.dataKey]}
+                                    // Set tabindex to -1 to prevent button focus
+                                    tabIndex={-1}
+                                    IconComponent={AddIcon}
+                                  >
+                                    {s.dataKey in sort && sort[s.dataKey]}
+                                  </TableSortLabel>
```

Comment:
You could simply & reduce code repetition here by removing the ternary checks and changing `TableSortLabel` to:

```
<TableSortLabel
   active={s.dataKey in sort}
   direction={sort[s.dataKey]}
   // Set tabindex to -1 to prevent button focus
   tabIndex={-1}
   IconComponent={!(s.dataKey in sort) && shiftDown ? AddIcon : undefined}
>
   {s.dataKey in sort && sort[s.dataKey]}
</TableSortLabel>
```